DNSExfiltration.md

```diff
@@ -30,6 +33,7 @@ Learn how to prevent DNS exfiltration from a GitHub Actions workflow.
       with:
         egress-policy: audit
     ```
+    For a seamless experience, StepSecurity recommends copying the full code from [here](https://github.com/step-security/supply-chain-goat/blob/main/sourcecodes/tut1/p3.md) and replacing the previous text entirely.
```

Comment:
I would recommend not taking the user to a different page. Why not just have the snippet in this page? It is not that long...

Reply:
okay sir, will do that